SAGE Journals.js

```diff
@@ -116,36 +117,35 @@ function scrape(doc, url) {
 					item.title += ': ' + subtitle.trim();
 				}
 			}
-			// The encoding of apostrophs in the RIS are incorrect and
-			// therefore we extract the abstract again from the website.
+			
+			// ubtue: extract translated and other abstracts from the different xpath
+			var ubtueabstract = ZU.xpathText(doc, '//article//div[contains(@class, "abstractSection")]/p');
+			var otherabstract = ZU.xpathText(doc, '//article//div[contains(@class, "tabs-translated-abstract")]/p');
 			var abstract = ZU.xpathText(doc, '//article//div[contains(@class, "abstractSection")]/p');
-			if (abstract) {
-				item.abstractNote = abstract.trim();
-			}
-
-			// ubtue: also add translated abstracts in 520
-			var translatedAbstract = ZU.xpathText(doc, '//article//div[contains(@class, "tabs-translated-abstract")]/p');
-			if (translatedAbstract) {
+			if (ubtueabstract && otherabstract) {
+				item.abstractNote = ubtueabstract;
 				item.notes.push({
-					note: "abs:" + translatedAbstract,
+					note: "abs:" + otherabstract.replace(/^Résumé/, ''),
 				});
+			} else if (ubtueabstract && !otherabstract) {
+				ubtueabstract = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "abstractInFull", " " ))]');
+				item.abstractNote = ubtueabstract;
+			} else if (!ubtueabstract && !otherabstract) {
+				ubtueabstract = ZU.xpathText(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "abstractInFull", " " ))]');
+				item.abstractNote = ubtueabstract;
+			} else {
+				item.abstractNote = abstract;
```

Comment:
A full example in a comment as to when which abstract should be used might be helpful to retrace the different cases

Reply:
As discussed: There are two else-if-branches `ubtueabstract && !otherabstract` and `!ubtueabstract && !otherabstract` that execute the same logic (extract `"abstractInFull"`). Thus this is equivalent to  `ubtueabstract || !ubtueabstract` and since this is alway true the code can be simplified to the single condition `!otherabstract`.